proto/scanners.proto

```diff
@@ -7,6 +7,16 @@ import "common.proto";
 
 service Scanners {
     rpc ReportJobsStatus(common.Empty) returns (JobReports);
+    rpc StopScan(StopScanRequest) returns (StopScanResult);
+}
+
+message StopScanRequest {
+	string scan_id = 1;
+}
+
+message StopScanResult {
+	bool status = 1;
+	string error = 2;
 }
```

Comment:
Should we have an `enum` instead of a `bool`?
I suggest we rename `string error` into `string description`

Reply:
I will rename the "error" to "description"
For the status, I think we will have only two possible values for the status -> 
true => Stop request was submitted successfully. 
false => Stop request was not submitted successfully.